src/features/game/lib/constants.ts

```diff
@@ -632,6 +632,7 @@ export const EMPTY: GameState = {
     Wood: new Decimal(50),
     Gold: new Decimal(10),
     Stone: new Decimal(10),
+    Carrot: new Decimal("334.119999999999991"),
```

Comment:
Can you revert this back before I merge?

Reply:
Done sir :)

Reply:
it was the only way to test the delivery in a local branch